torch/csrc/jit/ir.cpp

```diff
@@ -1218,6 +1218,30 @@ Node* Graph::createListUnpack(Value* v, size_t size) {
   return n;
 }
 
+Node* Graph::createDict(
+    const TypePtr& key_type,
+    const TypePtr& value_type,
+    std::unordered_map<Value*, Value*>& values) {
+
+  auto n = create(prim::DictConstruct, 1);
+  for (auto v : values) {
```

Comment:
nondeterministic IR warning! Iterating over unordered map will cause the IR to be different every time.  There is no need for the API to take a map, it is just being used as a list here anyway.